Packs/SecurityTrails/Integrations/SecurityTrails/README.md

```diff
@@ -1,4 +1,5 @@
 This integration provides API access to the SecurityTrails platform.
+This integration was integrated and tested with version xx of SecurityTrails
```

Comment:
Please add SecurityTrails current version.

Reply:
I don't believe SecurityTrails provide versioning. It's an online service. They simply have the /v1 service, so I'll add this.